lib/tapioca/dsl/compilers/active_record_relations.rb

```diff
@@ -679,6 +680,59 @@ def create_common_methods(common_relation_methods_module)
             end
           end
 
+          BATCHES_METHODS.each do |method_name|
+            case method_name
+            when :find_each
+              create_common_method(
+                "find_each",
+                common_relation_methods_module,
+                parameters: [
+                  create_kw_opt_param("start", type: "T.untyped", default: "nil"),
+                  create_kw_opt_param("end", type: "T.untyped", default: "nil"),
```

Comment:
```suggestion
                  create_kw_opt_param("finish", type: "T.untyped", default: "nil"),
```

https://github.com/rails/rails/blob/bd11e520a2e81d2b272a9812ca21ae199d74bf16/activerecord/lib/active_record/relation/batches.rb#L37